app/views/completed_applications/show.html.erb

```diff
@@ -8,7 +8,24 @@
     <%= render partial: 'return_notification',
                locals: { return_details: @crime_application.return_details } if @crime_application.returned? %>
 
-             <h1 class="govuk-heading-xl"><%= t(".heading.#{@crime_application.application_type}") %></h1>
+    <div class="govuk-!-margin-bottom-4">
+      <% if @crime_application.review_status.present? %>
+        <%= govuk_tag(text: t(@crime_application.review_status, scope: 'review_status'), colour: 'blue') %>
+      <% end %>
+      <% if @crime_application.decisions.present? %>
+        <%= govuk_link_to(t('.view_funding_decision'), '#funding_decisions', no_visited_state: true, class: 'govuk-!-margin-left-2') %>
+      <% end %>
+    </div>
+
+    <p class="govuk-body-l govuk-!-margin-top-5 govuk-!-margin-bottom-1">
+      <%= @crime_application.reference %>
+    </p>
+
+    <h2 class="govuk-heading-xl govuk-!-margin-bottom-4">
+      <%= @crime_application.applicant.full_name %>
+    </h2>
+
+    <h1 class="govuk-heading-xl"><%= t(".heading.#{@crime_application.application_type}") %></h1>
```

Comment:
just noticed h1 is after h2 here and perhaps the app type should be smaller in size than the name?